addons/website/i18n/website.pot

```diff
@@ -1684,6 +1684,11 @@ msgstr ""
 msgid "Email support"
 msgstr ""
 
+#. module: website
+#: model:ir.ui.view,arch_db:website.res_config_settings_view_form
+msgid "Enable billing on your Google Project"
+msgstr ""
```

Comment:
Is not this automatic now?

Reply:
Unless it is automatic since recently, only `*.po` are generated weekly.

Reply:
Oh? I don't know the difference with pot, what is it?

Reply:
{module}.pot is the template of terms to translate

regularly https://www.transifex.com scans github change and if there is a change eg. in `addons/website/i18n/website.pot` it will be added in the terms to translate on https://www.transifex.com/odoo/public/

Every Sunday, all translations change on https://www.transifex.com are automatically exported on files such as `addons/website/i18n/fr.po`.

But the *.pot are automatically updated for a version only once when a new version is released, so there is only in master where it should not be necessary to update *.pot when a new term is added (or eg. in some accounting localization we only use one language).

Reply:
Thanks for the explanation ! :)